### DISCUSSION TITLE
Livewire views not found after deployment

### DISCUSSION THREAD

**Discussion by ElijahSchrock** [livewire/livewire#5026] [1. Help]
After deployment to Forge, my Laravel Livewire site is throwing out a server error. I am using Digital Ocean as well. In the Forge site logs it says that my livewire view can't be found. The app works perfectly on local. Any ideas what to do?

I have attempted these commands php artisan commands
![image](https://user-images.githubusercontent.com/70502486/173217198-c6eb1ee8-11a8-4d4f-8429-caab8683f243.png)

Here is the site logs pt.1
![image](https://user-images.githubusercontent.com/70502486/173217203-65ff20df-3475-4b2b-92ab-30448d1c06cc.png)

and pt.2 
![image](https://user-images.githubusercontent.com/70502486/173217219-696f5852-1042-47f2-a6a1-30545b9956c2.png)

I have followed the path of my files and I can confirm there aren't any spelling errors, and to my knowledge, the names of my files are not an issue, since I have followed conventional php artisan make:livewire folder.FooBar as the documentation states. One thing I might think was that I changed the names and rearranged the folders maybe 20 commits before my deployment commit, but I can't see that affecting anything. Please if you have any suggestions I'd love the help

Here is my github [Repository](https://github.com/ElijahSchrock/Syphon)

**Comment by joshhanley** [comment]
@ElijahSchrock I've not seen that before, so not sure I can be of much help. But can you try deploying your old commit before all the changes and see if it starts working again?

**Reply by ElijahSchrock** [reply]
hi thank you for you help! i actually figured it out. my local had correct folder names but my master didn't catch the commit to update the folder names correctly. so the whole problem was just case sensitive issue.

**Reply by joshhanley** [reply]
@ElijahSchrock ah right, glad you got it sorted 🙂 Do you want to add an answer below with some details and mark it as answered, incase others run into the same issue in the future?

**Reply by ElijahSchrock** [reply]
definitely will, traveling home from vacation but I will have one up by end of day. Thank you for your help :)

**Reply by joshhanley** [reply]
@ElijahSchrock ok thanks! No worries 🙂
